CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # https://help.github.com/en/articles/about-code-owners
 # Documentation: https://github.com/hmcts/cnp-flux-config/blob/master/CODEOWNERS.md
 
-CODEOWNERS @hmcts/platform-engineering @hmcts/devops
+CODEOWNERS @hmcts/platform-engineering @hmcts/devops @hmcts/ccd-demo
```

Comment:
don't do this, just add one for ccd-demo

Reply:
```suggestion
CODEOWNERS @hmcts/platform-engineering @hmcts/devops
```

Reply:
deleted